app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -196,6 +198,8 @@
     private boolean isVisibleToUser;
     private MapboxMap.OnCameraMoveListener cameraMoveListener;
     private fr.free.nrw.commons.location.LatLng lastFocusLocation;
+    private ScaleBarPlugin scaleBarPlugin;
```

Comment:
these don't need to be fields I think, local variables should be fine

Reply:
Done 👍  
@macgills anything else?

Reply:
I think that is it from me, push away. Good work